plugins/linguist-backend/src/index.ts

```diff
@@ -20,6 +20,7 @@
  * @packageDocumentation
  */
 
+export * from './processor';
```

Comment:
technically this could have gone in a new `plugins/catalog-backend-module-linguist`, but that can come in a future update down the line :)